src/app/qgsvectorlayerproperties.cpp

```diff
@@ -293,7 +293,6 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   );
 
   QString myStyle = QgsApplication::reportStyleSheet();
-  myStyle.append( QStringLiteral( "body { margin: 10px; }\n " ) );
```

Comment:
I think it should be rolled back - the stylesheet is used in a number of space constrained places, like the expression builder help and browser dock widget info panel, so I don't think it's a good move to always add these margins.